### DISCUSSION TITLE
arch firefox + gui

### DISCUSSION THREAD

**Discussion by p20sami** [courses-ionio/hci#1628] [Announcements]
Edit:
Καλησπέρα σας, όπως αναφέρει και ο τίτλος προέκυψαν κάποια προβλήματα στη γραμμή εντολών. Συγκεκριμένα, εγκατέστησα τον firefox ώστε να έχω πρόσβαση στο asciinema κλπ πέραν του terminal, αλλά όταν προσπαθώ να το ανοίξω εμφανίζει error: no display environment variable specified. Κατά την εγκατάσταση δεν εγκατέστησα κάποιο gui ή wm, το άφησα σαν καθαρό τερματικό. Θα χρειαστεί να εγκαταστήσω κάποιο wm ή gui ώστε να μπορώ να ανοίγω browser απο το τερματικό?
Σας ευχαριστώ εκ των προτέρων.

**Comment by p16koub1** [comment]
Δεν είμαι σίγουρος, αλλά νομίζω πως πρέπει να εγκαταστήσεις το [xorg](https://wiki.archlinux.org/title/xorg).
Για γραφικό περιβάλλον, μπορείς να εγκαταστήσεις τo [dwm](https://dwm.suckless.org/), μπορείς να δεις και αυτόν τον [οδηγό εγκατάστασης ](https://www.youtube.com/watch?v=jD8BtmMK0do&t=308s&ab_channel=SavvyNik) αν αντιμετοπίσεις κάποιο πρόβλημα. Επίσης μπορείς να βρεις και έτοιμα builds ([LukeSmithxyz-dwm](https://github.com/lukesmithxyz/dwm), [distro-tube-dwm](https://gitlab.com/dwt1/dwm-distrotube))

**Reply by p20sami** [reply]
Ευχαριστώ για την απάντηση, όντως νομίζω πως πρέπει να γίνει η εγκατάσταση του xorg, αλλά πάντα βγάζει xorg-server-21.1.1-2-x86_64.pkg.tar.zst failed to download, το ίδιο και με sddm κλπ

**Comment by epidrome** [comment]
firefox dependencies: https://archlinux.org/packages/extra/x86_64/firefox/

gui topic #1621 

arch issues: https://bbs.archlinux.org
